doc/nrf/libraries/others/dm.rst

```diff
@@ -0,0 +1,75 @@
+.. _mod_dm:
+
+Distance Measurement
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Distance Measurement module provides an integration of :ref:`nrfxlib:nrf_dm` into |NCS|.
+
+Overview
+********
+
+The Distance Measurement module is independent of the communication protocol used.
+The :ref:`nrfxlib:mpsl` is used to access the radio module which is required by the :ref:`nrfxlib:nrf_dm` library.
```

Comment:
the timeslot is not directly required by the library in nrfxlib.

Reply:
Modified.